rust/metaplex/program/Cargo.toml

```diff
@@ -13,14 +13,14 @@ no-entrypoint = []
 test-bpf = []
 
 [dependencies]
-spl-auction = { path = "../../auction/program", features = [ "no-entrypoint" ] }
+spl-auction = { path = "../../auction/program", features = [ "no-entrypoint" ], version = "0.0.1" }
```

Comment:
So inside the repo we would still want to be pointing at relative so we can do easy code edits